src/templates/partials/header.html

```diff
@@ -0,0 +1,40 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="{% url '' %}">
```

Comment:
I can't run the application with the empty url (`% url '' %`), could you either remove or replace with `href="#"` until we have URLs?